examples/basic-express-example/server.js

```diff
@@ -1,78 +1,87 @@
+'use strict';
```

Comment:
Changed this file to print an example cURL command and not start express server endpoints until sync has finished initialising.